ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -516,7 +497,7 @@ void pdo_sqlite_create_function_internal(INTERNAL_FUNCTION_PARAMETERS)
 
 	ZEND_PARSE_PARAMETERS_START(2, 4)
 		Z_PARAM_STRING(func_name, func_name_len)
-		Z_PARAM_FUNC(fci, fcc)
+		Z_PARAM_FUNC_NO_TRAMPOLINE_FREE(fci, fcc)
```

Comment:
This would leak if you have a trampoline and one of the follow-up arguments fails the ZPP check I think

Reply:
Oh, indeed. This will leak.

Reply:
Fixed it https://github.com/php/php-src/pull/14059/commits/82c2bac5f892978c2ad59b3dac627b60e0aa119f